resources/serverless/charts/docker-registry/templates/job.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if eq .Values.storage "filesystem" }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "docker-registry.name" . }}-image-gc
+spec:
+  schedule: "42 * * * *"
```

Comment:
Why `42`?

Reply:
It's the answer to life, the universe and everything!

Also, I tend to cron things away from common time-slices to avoid load spikes.

Reply:
Could you add comment above the `schedule`?
It's clever solution.